### PR TITLE
test(lwq): test maqac.py & polish annotations in maqac.py

### PR DESCRIPTION
**Pull request by Hcnaeg**
## Check List

- [ ] merge the latest version source branch/repo, and resolve all the conflicts
- [ ] pass style check
- [ ] pass all the tests

### CODE REVIEW

**Review by PaParaZz1 — APPROVED** [review]

File: ding/torch_utils/network/tests/test_resnet.py [hunk 1]
```diff
@@ -10,3 +13,93 @@ def test_resnet18():
...
+    model.init_weights(zero_init_last_bn=zero_init_last_bn)
+    inputs = torch.randn(4, 3, 224, 224).requires_grad_(True)
+    outputs = model(inputs)
+    assert outputs.shape == (4, num_classes if num_classes > 0 else 512)
+    mse_loss = torch.nn.MSELoss()
+    target = torch.randn(outputs.shape)
+    loss = mse_loss(outputs, target)
+    assert inputs.grad is None
+    loss.backward()
+    assert isinstance(inputs.grad, torch.Tensor)
```
Comment (kxzxvbk) [inline ding/torch_utils/network/tests/test_resnet.py:56]: I think it is strange to examine the gradient of input. Why not check the gradient of the model?

Comment (PaParaZz1) [reply ding/torch_utils/network/tests/test_resnet.py:56]: Good point!
